### PR TITLE
fix the JVM memory issue for people with low ram

### PR DESCRIPTION
**Pull request by Hiro420**
is to fix the JVM memory issue for some people with low amount of ram

## Description

Please carefully read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md) before making any pull requests.
And, **Do not make a pull request to merge into stable unless it is a hotfix. Use the development branch instead.**
## Type of changes

- [x] Bug fix
- [ ] New feature 
- [ ] Enhancement
- [ ] Documentation

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] My pull request is unique and no other pull requests have been opened for these changes
- [x] I have read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md)
- [x] I am responsible for any copyright issues with my code if it occurs in the future.

### CODE REVIEW

**Comment by Birdulon** [comment]
I'm not sure what issue this is supposed to fix, the default value for that is 512MB so raising it to 4096MB is likely to be worse for low RAM.
https://docs.gradle.org/5.6.4/userguide/build_environment.html#sec:configuring_jvm_memory

**Comment by KingRainbow44** [comment]
More specifically, this would only help people when *building* Grasscutter. It's pretty much assumed that if you're building Grasscutter that you either: A: have enough ram or B: know how to allocate more JVM heap in your system's **global** `gradle.properties` file.

TLDR: Instead of proposing this *directly* to the repo, it's a better idea to instead add this to documentation.
